quiz_system/key.py

```diff
@@ -0,0 +1,5 @@
+class Key:
+
+    def __init__(self, key_id, key_text):
+        self.key_id = key_id
+        self.key_text = key_text
```

Comment:
I would suggest creating following hierarchies for classes with respective responsibilities:

- FileReader (does file i/o, loads files given file path)
- Quiz System  (poses questions, input answers, show results)
- Question   (represents questions, hold answer, check if correct)